src/app/components/core/developer/additional-services/additional-service-modal/additional-service-modal.component.ts

```diff
@@ -45,6 +39,9 @@ export class AdditionalServiceModalComponent {
   state$ = new BehaviorSubject<any>(null);
   service = <Service>{};
 
+  serviceInfo = { added: false, type: '', isEnabled: false, schedule_process: '', process: '', package: '',
```

Comment:
What is the different b/w schedule_process and  process?

Reply:
`schedule_process` is the processName which we get in the /schedule API response like- notification_c, bucket_storage_c etc. While `process` is having values like `notification`, `bucket` etc which we get in the response of /installed API.